Index: src/CoreGame/MapComponent/Tile.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package CoreGame.MapComponent;\r\n\r\nimport CoreGame.Data.Enums.Collision;\r\n\r\nimport java.awt.image.BufferedImage;\r\n\r\npublic class Tile\r\n{\r\n    public BufferedImage image;\r\n    public Collision collision = Collision.NoCollision;\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/CoreGame/MapComponent/Tile.java b/src/CoreGame/MapComponent/Tile.java
--- a/src/CoreGame/MapComponent/Tile.java	(revision 46cea39f2d7b9d6e78935308bcf5b88ae50875ca)
+++ b/src/CoreGame/MapComponent/Tile.java	(date 1734982034168)
@@ -6,6 +6,6 @@
 
 public class Tile
 {
-    public BufferedImage image;
+    public BufferedImage bufferedImage;
     public Collision collision = Collision.NoCollision;
 }
